pages/tools.md

```diff
@@ -14,6 +14,10 @@ secondary_nav:
 ---
 ## Agency data inventories
 
+[How to get your Open Data on Data.gov]({{ site.baseur }}{% link tools/how-to-get-your-open-data-on-datagov.md %}).
```

Comment:
Should be site.baseurl, right?

Reply:
:frowning_face: added an issue to improve testing here...

Reply:
Pushed a fix.